### PR TITLE
Add condition based on forkedRepos length

### PR DESCRIPTION
**Pull request by josenriagu**
this Pull Request fixes the error caused by calling **forkedRepos** variable before initialization. It also adds a condition that prevents subsequent functions from executing if there exists no forkedRepos.

### CODE REVIEW

**Comment by birdcar** [comment]
@josenriagu it looks like there might have been conflicts introduced by @erin-koen in #9. 

Could you resolve the merge conflicts on this branch so I can review your code? Thank you!

**Comment by josenriagu** [comment]
> @josenriagu it looks like there might have been conflicts introduced by @erin-koen in #9.
> 
> Could you resolve the merge conflicts on this branch so I can review your code? Thank you!

Sure buddy. Let me look into it

**Comment by josenriagu** [comment]
@nickcannariato the conflicts have been resolved now.

File: spoon.js [hunk 1]
```diff
@@ -24,24 +24,29 @@ axios.interceptors.request.use(config => {
...
-  await renameForks(forkedRepos);
-  console.log("Renaming complete.");
-  console.log("Generating new repos...");
-  await genRepos(forkedRepos);
-  console.log("New repos complete.");
-  console.log("Importing data from forks to new repos...");
-  await importData(forkedRepos);
-  console.log("Import complete!");
+  if (forkedRepos.length === 0) {
+    console.log('You currently have no forked repos, here\'s a pony!')
```
Comment (birdcar) [inline spoon.js:34]: This made me laugh, love the tone! 😆

**Review by birdcar — APPROVED** [review]
Thank you for your contribution! 🚀

**Pull request merged**